pom.xml

```diff
@@ -170,8 +170,8 @@
     <!-- for now, not running scalafmt as part of default verify pipeline -->
     <scalafmt.skip>true</scalafmt.skip>
     <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
-    <fasterxml.jackson.version>2.13.2</fasterxml.jackson.version>
-    <fasterxml.jackson.databind.version>2.13.2.1</fasterxml.jackson.databind.version>
+    <fasterxml.jackson.version>2.13.3</fasterxml.jackson.version>
+    <fasterxml.jackson.databind.version>2.13.3</fasterxml.jackson.databind.version>
```

Comment:
I add this property because of SPARK-38665, and it can be removed because all jackson jars share same version again.

Reply:
If you look at the history, SPARK-38665 is not the only one did that. `databind` is frequently separated and merged back repeatedly. :) That's the reason why I decided not to remote that property back in this PR. 

```
[SPARK-38665][BUILD] Upgrade jackson due to CVE-2020-36518
[SPARK-33695][BUILD] Upgrade to jackson to 2.10.5 and jackson-databind to 2.10.5.1
[SPARK-28728][BUILD] Bump Jackson Databind to 2.9.9.3
```